clarity/evaluator/haaqi/haaqi.py

```diff
@@ -147,3 +147,31 @@ def haaqi_v1(
     raw = [mel_cepstral_high, basilar_membrane_sync5, d_loud, d_norm]
 
     return combined_model, nonlinear_model, linear_model, raw
+
+
+def compute_haaqi(
```

Comment:
Thanks for moving this to the main `pyclarity` code, its going to make reusing it in the future much easier.

Is it possible to write a really quick test for this so that we maintain test coverage and can ensure that any future changes do not break the functionality?